asapdiscovery-dataviz/asapdiscovery/dataviz/_gif_blocks.py

```diff
@@ -99,19 +120,33 @@
 }
 
 # MERS
-pocket_dict_mers = {
+pocket_dict_mers_mpro = {
     "subP1": "143+144+145+146+147+148+166+175",
     "subP1_prime": "25+26+27",
     "subP2": "41+49+54",
     "subP3_4_5": "168+169+170+171+192+193+194+195",
     "sars_unique": "25+49+145+167+171+172+184+189+191+193+194",
 }
 
-color_dict = {
+# MAC1
+pocket_dict_mers_mpro = {
```

Comment:
Is this meant to be `pocket_dict_sars2_mac1`?

```suggestion
pocket_dict_sars2_mac1 = {
```

Reply:
thanks!